timesketch/lib/analyzers/interface.py

```diff
@@ -48,11 +48,13 @@ def get_yaml_config(file_name):
         an empty dict if the file is not found or YAML was unable
         to parse it.
     """
-    root_path = os.path.join(os.path.sep, 'etc', 'timesketch')
-    if not os.path.isdir(root_path):
+    # root_path = os.path.join(os.path.sep, 'etc', 'timesketch')
+    config_path = os.path.join(os.path.sep, 'usr', 'local', 'src', 'timesketch', 'config')
```

Comment:
I had to change this to get it to work for me; I'm not sure if it's specific to my instance (or if this is used anywhere else).

Reply:
Ack, reverted.